libdevcore/Result.h

```diff
@@ -54,14 +62,13 @@ class Result
 	/// @returns the error message (can be empty).
 	std::string const& message() const { return m_message; }
 
-	/// Merges _other result into this using the _merger
-	/// and appends the error messages.
-	/// Meant to be called with logical operators like
-	/// logical_and, etc.
+	/// Merges _other into this using the _merger
+	/// and appends the error messages. Meant to be called
+	/// with logical operators like logical_and, etc.
 	template<typename F>
-	void merge(Result<ResultType> const& _other, F merger)
+	void merge(Result<ResultType> const& _other, F _merger)
```

Comment:
You applied the fix to the wrong commit

Reply:
I was lazy and you're right. I will change it.

Reply:
Has this been resolved?

Reply:
Yes :)